kiva/trait_defs/kiva_font_trait.py

```diff
@@ -10,181 +10,7 @@
 """ Trait definition for a wxPython-based Kiva font.
 """
 
-import logging
+from enable.trait_defs.kiva_font_trait import KivaFont as _KivaFont
```

Comment:
Are you sure we want to import from `enable` here?

Reply:
Yes, I _think_ so.  It's preliinary to getting rid of `kiva.trait_defs`.

As long as we don't have circularity of imports it should be fine.